### DISCUSSION TITLE
Can renovate append a new version to list of versions instead of replacing in a file?

### DISCUSSION THREAD

**Discussion by valdisrigdon** [renovatebot/renovate#27783] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Our use case is that we re-build upstream images and apply common hardening to them. 

When we detect a new version, say v2, we want to add that to a list of items in a configuration files. Then the build process would build v1 and v2 of the upstream image until we rollout v2 everywhere, then we can remove v1. The versions we produce would be independent of the upsteam version number.

I would assume that I'd need the regex manger to do this -- is there a way to configure it in such a way?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Short version is that: no, Renovate natively has on concepts of "sets" where it can reduce, update or expand a set of dependencies. You might be able to hack regex manager into it, but I'm not sure it will be possible.

I would like to support this concept one day, for example allowing users to have Renovate maintain the list of LTS node versions, but it would not be a small code change.
